tutorials/AFQ_publish_tutorials.m

```diff
@@ -0,0 +1,36 @@
+function AFQ_publish_tutorials(remotename)
+% Publish all the AFQ tutorials as html pages push to gh-pages
+%
+% AFQ_publish_tutorials(remotename)
+%
+% inputs:
+% remotename = string for a git remote repository. Default is 'origin'
+%
+% Written by JDY, Feb, 2016
+
+if ~exist('remotename', 'var') || isempty(remotename)
+    remotename = 'origin';
+end
+afqdir = AFQ_directories;
+tdir = fullfile(afqdir,'tutorials');
+
+% List of tutorials to publish
+tlist = {'AFQ_example.m'};
```

Comment:
Maybe look at the file-system to see what files are there and run this over all of the m files, unless a specific input is given? Otherwise, when you add a tutorial, you will have to remember to edit this as well.